### DISCUSSION TITLE
Does this work under windows?

### DISCUSSION THREAD

**Discussion by alkampfergit** [montudor/action-zip#5] [Q&A] [answered]
Hi,

actually the action does not seems to work in windows machines, am I right?

![image](https://user-images.githubusercontent.com/358545/76701505-0c0ba600-66c2-11ea-8ecb-db463672bbc4.png)

**Comment by montudor** [comment, marked as answer]
That's correct - this action won't work under windows since it works using Docker.

Sorry about that. I have added a note to the README.

Note: GitHub Actions didn't support Windows when I created this action. I will have to look into it and see if I can add support at some point.

**Comment by alkampfergit** [comment]
Ok thanks, I just wanted to be sure that I'was not missing something obvious :)
